roles/windows/common/tasks/main.yml

```diff
@@ -1,9 +1,21 @@
 ---
-- name: Install windows updates
-  include_tasks: 'roles/windows/windows_updates/tasks/main.yml'
+# - name: Install windows updates
```

Comment:
Curious why this is being commented out?

Reply:
It was breaking the image builds, I didnt realize it was still commented out when I made the PR, let me uncomment it and see if it breaks still